frontend/src/component/project/Project/Project.tsx

```diff
@@ -64,6 +66,43 @@ interface ITab {
     isEnterprise?: boolean;
 }
 
+const CircleContainer = styled('div')(({ theme }) => ({
+    position: 'absolute',
+    width: theme.spacing(2.5),
+    height: theme.spacing(2.5),
+    display: 'grid',
+    placeItems: 'center',
+    borderRadius: '50%',
+
+    background: theme.palette.background.alternative,
+    color: theme.palette.primary.contrastText,
+    fontSize: theme.typography.body2.fontSize,
+
+    // todo: revisit these values later
+    top: 10,
+    right: 0,
+    [theme.breakpoints.down('md')]: {
+        top: 2,
+    },
```

Comment:
I wanna make sure these values don't break in more scenarios, but I'll have a look at it when I actually hook it up.